src/ConstantSum/ConstantSumSolver.sol

```diff
@@ -59,7 +59,7 @@ contract ConstantSumSolver {
         SimulateSwapState memory state;
 
         if (swapXIn) {
-            state.deltaLiquidity = amountIn.mulWadUp(poolParams.swapFee);
+            computeSwapDeltaLiquidity(amountIn, poolParams, true);
```

Comment:
This is a `pure` function that returns a value, but the return value isn't being used at all. Looks like `state.deltaLiquidity` is never set in this branch.

Reply:
Good point, I think it's an artefact from the past, let's remove it.

Reply:
The function call was deleted here https://github.com/primitivefinance/DFMM/pull/119/commits/016b81a75b3fc8c300a91d004690b2abe38f8ae2.